go/interpreter/lfvm/gas_test.go

```diff
@@ -59,3 +59,171 @@ func TestGas_CallGasCalculation(t *testing.T) {
 		})
 	}
 }
+
+// --- SStore ---
+
+func TestGas_getDynamicCostsForSstore_exhaustive(t *testing.T) {
+	// This test exhaustively checks the computation of the dynamic gas costs for
+	// the SSTORE instruction by enumerating every possible input combination
+	// and comparing the result with the specification found on
+	// https://www.evm.codes.
+
+	// The specification found on https://www.evm.codes is provided in the form
+	// of a python code snippet. The following function is a direct translation
+	// of the python code to Go, parameterized to allow for easy testing of
+	// different revisions.
+	makeSpec := func(create, update, touch tosca.Gas) func(s storageStateExample) tosca.Gas {
+		return func(s storageStateExample) tosca.Gas {
+			// source: https://www.evm.codes
+			gas := tosca.Gas(0)
+			if s.value == s.current {
+				gas = touch
+			} else if s.current == s.original {
+				if s.original == 0 {
+					gas = create
+				} else {
+					gas = update
+				}
+			} else {
+				gas = touch
+			}
+			return gas
+		}
+	}
+
+	specs := map[tosca.Revision]func(storageStateExample) tosca.Gas{
+		// source: https://www.evm.codes/?fork=istanbul
+		tosca.R07_Istanbul: makeSpec(20000, 5000, 800),
+		// source: https://www.evm.codes/?fork=berlin
+		tosca.R09_Berlin: makeSpec(20000, 2900, 100),
+	}
+
+	// All other revisions are based on Berlin.
+	specs[tosca.R10_London] = specs[tosca.R09_Berlin]
+	specs[tosca.R11_Paris] = specs[tosca.R10_London]
+	specs[tosca.R12_Shanghai] = specs[tosca.R11_Paris]
+	specs[tosca.R13_Cancun] = specs[tosca.R12_Shanghai]
```

Comment:
if the comment says `all other revisions are base don berlin`  why not just do this ? 
```suggestion
	specs[tosca.R10_London] = specs[tosca.R09_Berlin]
	specs[tosca.R11_Paris] = specs[tosca.R09_Berlin]
	specs[tosca.R12_Shanghai] = specs[tosca.R09_Berlin]
	specs[tosca.R13_Cancun] = specs[tosca.R09_Berlin]
```

Reply:
I prefer to update the comment to state that they inherit the specification from their predecessor -- which is more accurate.